spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/SwaggerDemoApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.swagger_responses;
```

Comment:
@apeterlic although Java naming convention allows underscore for package naming we avoid them, so please rename it, you can just use swaggerresponses.